### PR TITLE
Add support for plaque AD

### PR DESCRIPTION
**Pull request by MaiTheSan**
## Description
Add support for Plaque AD TKL PCB

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/23359

## Checklist

- [ ] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by Cipulot** [comment]
Adding `Pending QMK merge` label.

**Comment by dunk2k** [comment]
Alpha keys and modifier keys don't have colours assigned 🙈

File: v3/asmodus/plaquead/plaquead.json [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+  "vendorId": "0x5A50",
+  "productId": "0x0875",
+  "matrix": {
+    "rows": 6,
+    "cols": 17
+  },
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_rgblight"],
+  "layouts": {
+    "labels": [
```
Comment (dunk2k) [inline v3/asmodus/plaquead/plaquead.json:12]: missing F12 function row (layout)

**Comment by Cipulot** [comment]
@dunk2k no point in submitting reviews at this time if the board isn't even merged in QMK main as per requirements.

**Pull request closed without merging**